### PR TITLE
Use atom for webview ref instead of storing it in re-frame's db

### LINKED ISSUE #10189: App crash when closing WebView
# Bug Report

## Problem
App crashes when WebView is closed via top-left cross icon. 
**NOTE:** In develop builds (nightly 19th March) `...-univeral.apk` it may not crash from the first time but if to open-close up to 20 times the same Dapp it may crash.

However with the builds from https://github.com/status-im/status-react/pull/10090 (https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200318-154045-e28f8e-pr-universal.apk) on Android 9 it crashes with the same stacktrace in logcat from the very first attempt (just open and close DApp).
And also develop nightly builds `-x86.apk` crashing all the time when autotests are running on emulators. 

#### Expected behavior

Can proceed using app after closing webview
#### Actual behavior

App crashes and closes.  In logcat: 

```

--------- beginning of crash
03-19 16:22:40.911 15236 15253 F libc    : Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x20 in tid 15253 (RenderThread), pid 15236 (status.ethereum)
03-19 16:22:40.976  3305  3305 I SurfaceFlinger: Display 0 HWC layers:
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:     type    |    handle    | flag |  format   |   source crop (l,t,r,b)    |         frame       | name 
03-19 16:22:40.976  3305  3305 I SurfaceFlinger: ------------+--------------+------+-----------+----------------------------+---------------------+------
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:      Device | 0x7009546020 | 0082 | RGBA_8888 |   0.0    0.0 1080.0 2220.0 |    0    0 1080 2220 | im.status.ethereum/im.status.ethereum.MainActivity[15236]#1
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:      Device | 0x7015e536a0 | 0000 | RGBA_8888 |   0.0    0.0 1080.0   72.0 |    0    0 1080   72 | StatusBar[4142]#0
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:      Device | 0x7015e54580 | 0000 | RGBA_8888 |   0.0    0.0 1080.0  144.0 |    0 2076 1080 2220 | NavigationBar[4142]#0
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:      Device | 0x7015a4a340 | 0000 | RGBA_8888 |   0.0    0.0   99.0   99.0 |  513  718  612  817 | Sprite#3
03-19 16:22:40.976  3305  3305 I SurfaceFlinger:      Device | 0x7015a4a180 | 0000 | RGBA_8888 |   0.0    0.0   99.0   99.0 |  554  730  653  829 | Sprite#8
03-19 16:22:40.976  3305  3305 I SurfaceFlinger: 
03-19 16:22:40.985  3827  3988 D InputReader: Input event(7): value=0 when=39976.242501
03-19 16:22:40.985  3827  3988 D InputReader: Input event(7): value=0 when=39976.242501
03-19 16:22:40.985  3827  3988 I InputReader: Touch event's action is 0x1 (deviceType=0) [pCnt=1, s=] when=39976.242501
03-19 16:22:40.986  3827  3987 I clp-JNI : Add pinfo.  (channel) f8bdfec im.status.ethereum/im.status.ethereum.MainActivity (server) (pkg) AppWindowToken{61411f4 token=Token{31b7c7 ActivityRecord{497d506 u0 im.status.ethereum/.MainActivity t1903}}} 15236 10345 (action) 1

```

#### Notes
https://github.com/status-im/status-react/pull/9945 seems the PR introduced that bug

### Reproduction

1) Open Status and create new account
2) Open Browser -> dap.ps (or any other Dapp or website)
3) Tap top-left `X` cross icon
Actual result: App crashes with `03-19 14:15:34.466 24256 24272 F libc    : Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x20 in tid 24272 (RenderThread), pid 24256 (tus.ethereum.pr)
`

### Additional Information
- Status version: develop nightly 19th of March
- Operating System: Android and iOS

#### Logs
(Full crash log: 

[crashNightly11.txt](https://github.com/status-im/status-react/files/4355148/crashNightly11.txt)

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #10189 

Kudos to @Ferossgp for finding the root cause of crashes and detailing it [here](https://github.com/status-im/status-react/issues/10189#issuecomment-601735469), this is essentially an implementation of a proposed fix.

### CODE REVIEW

**Comment by Serhy** [comment]
With the latest build https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200323-165257-976b2a-pr-universal.apk I don't have any crashes with webview. When it closes it does it smoothly without blinking like before. Confirmed on Android 9 (where 100% crash were) and Android 8, iOS13.3
Two failed tests failed in develop as well (seems like an issue with app). Anyway not this PR bug.
Well done, @siphiuel and @Ferossgp 

Waiting for review before merge...

**Review by Ferossgp — APPROVED** [review]

**Pull request merged**
